docusaurus/docs/cloud/account/settings.md

```diff
@@ -7,13 +7,13 @@ canonicalUrl: https://docs.strapi.io/cloud/account/settings.html
 
 # Account profile
 
-The *Profile* page enables you to manage your account details and preferences. It is accessible by clicking on your profile picture, on the top right hand corner of the interface, and **Profile**. There are 3 tabs available in the *Profile* interface: *General*, *Preferences* and *Billing*. 
+The *Profile* page enables you to manage your account details and preferences. It is accessible by clicking on your profile picture, on the top right hand corner of the interface, and **Profile**. There are 3 tabs available in the *Profile* interface: *General*, *Preferences* and *Billing*.
 
 ## General
 
 The *General* tab enables you to edit the following details for your account profile:
 
```

Comment:
@pwizla Will work on adding the **Connected accounts** section ?

Reply:
I'm not sure I understand your question, @AtallahFatma. Where is this section in the Cloud dashboard?

Reply:
Hey @pwizla , here's the [design](https://www.figma.com/file/u9IfEz5nhUwoIt5nsvT4Zh/%5BEXPLORATION%5D-Cloud---Settings?type=design&node-id=838%3A3907&mode=design&t=PS3tLi4tjulrAFIk-1) that I'm talking about

Reply:
Oh, thanks! I wasn't made aware of this change. Will this be part of this week's release? Is it already on staging so I can take some screenshots?

Reply:
It's not yet on staging and for the moment, the release is planned for Wednesday until further notic.

Reply:
Thanks for the update :) I'll modify the text content later today and will add the screenshot as soon as it's ready on staging.

Reply:
Update done ✅